cmake/Ekat.cmake

```diff
@@ -1,3 +1,6 @@
+# Where ekat's tpls live
```

Comment:
Is this true? It seems like it's where EKAT's CMake files live.

Reply:
Yeah, that's what I meant: ekat's cmake scripts for tpls. I'll fix it.